.gitignore

```diff
@@ -2,6 +2,7 @@
 *.js
 *.js.map
 src/*.d.ts
+lib/
```

Comment:
I usually prefer `dist/` since this is not really a library.

Reply:
I updated it to use `dist/`